plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerConnectorTest.java

```diff
@@ -52,6 +46,8 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
             case SUPPORTS_PREDICATE_PUSHDOWN_WITH_VARCHAR_INEQUALITY:
                 return false;
 
+            case SUPPORTS_AGGREGATION_PUSHDOWN_STDDEV:
+            case SUPPORTS_AGGREGATION_PUSHDOWN_VARIANCE:
```

Comment:
define them in separate code group, don't glue them to `case SUPPORTS_JOIN_PUSHDOWN`
```suggestion
            case SUPPORTS_AGGREGATION_PUSHDOWN_VARIANCE:
                return true;
                
```